browser.js

```diff
@@ -46,7 +46,9 @@ function useColors() {
 
   // is webkit? http://stackoverflow.com/a/16459606/376773
   // document is undefined in react-native: https://github.com/facebook/react-native/pull/1632
-  return (typeof document !== 'undefined' && 'WebkitAppearance' in document.documentElement.style) ||
+  // document.documentElement is null in Electron preload scripts
+  return (typeof document !== 'undefined' && document.documentElement
+          'WebkitAppearance' in document.documentElement.style) ||
```

Comment:
wondering if you meant to add an `&&` to the end of the null pointer check `document.documentElement`?  This looks like a syntax error